crm/track_leads/lead_scoring.rst

```diff
@@ -20,59 +20,58 @@ To use scoring, install the free module *Lead Scoring* under your
+You now have a new tab in your *CRM* app called *Leads*
 where you can manage your scoring rules.
 
-Here's an example for a Canadian lead, you can modify for whatever
-criteria you wish to score your leads on. You can add as many criterias
-as you wish.
+Here's an example of a Canadian lead. You can modify your rules for whatever
+criteria you want to score leads on. You can add as many criteria
+as you would like.
 
 .. image:: media/lead_scoring02.png
    :align: center
 
-Every hour every lead without a score will be automatically scanned and
-assigned their right score according to your scoring rules.
+Every lead without a score will automatically be scanned every hour and
+assigned the correct score according to your configured scoring rules.
 
 .. image:: media/lead_scoring03.png
    :align: center
 
 Assign leads
 ============
 
-Once the scores computed, leads can be assigned to specific teams using
-the same domain mechanism. To do so go to :menuselection:`CRM --> Leads Management --> Team Assignation`
-and apply a specific domain on each team. This domain can include scores.
+Once the score is computed, leads can be assigned to specific teams using
+the same domain mechanism. To do so go to :menuselection:`CRM --> Leads --> Team Assignment`
+and apply a specific domain to each team. This domain can include scores.
 
 .. image:: media/lead_scoring04.png
    :align: center
 
-Further on, you can assign to a specific vendor in the team with an even
-more refined domain.
+If you want more granularity, you can assign leads to a specific salesperson on the team using
+further refined domains.
 
-To do so go to :menuselection:`CRM --> Leads Management --> Leads Assignation`.
+To do so go to :menuselection:`CRM --> Leads --> Leads Assignment`.
 
 .. image:: media/lead_scoring05.png
    :align: center
 
 .. note::
-   The team & leads assignation will assign the unassigned leads
-   once a day.
+   The team & leads assignments will assign the unassigned leads
+   once per day.
 
 Evaluate & use the unassigned leads
 ===================================
 
-Once your scoring rules are in place you will most likely still have
-some unassigned leads. Some of them could still lead to an opportunity
-so it is useful to do something with them.
+Once your scoring rules are in place, you likely will still have
+some unassigned leads. Some of these leads could still transition into an opportunity, so
+ensuring they are handled is both useful and important.
 
-In your leads page you can place a filter to find your unassigned leads.
+On your leads overview page, you filter for your unassigned leads.
 
 .. image:: media/lead_scoring06.png
    :align: center
 
-Why not using :menuselection:`Email Marketing` or
-:menuselection:`Marketing Automation` apps to send a mass email to
-them? You can also easily find such unassigned leads from there.
+You can also easily find unassigned leads by using the :menuselection:`Email Marketing` or
```

Comment:
You could use a [note](https://www.odoo.com/documentation/user/14.0/contributing/documentation/rst_cheat_sheet.html#note) alert block here. - to spice up your doc a tiny bit :)